src/scenes/Projects/ProjectList.tsx

```diff
@@ -43,11 +40,17 @@ export const ProjectList: FC = () => {
         {data?.projects.total} Projects
       </Typography>
       <Grid container direction="column" spacing={2}>
-        {data?.projects.items.map((item) => (
-          <Grid item key={item.id}>
-            <ProjectListItemCard {...item} />
-          </Grid>
-        ))}
+        {data
+          ? data?.projects.items.map((item) => (
+              <Grid item key={item.id}>
+                <ProjectListItemCard loading={loading} {...item} />
+              </Grid>
+            ))
+          : _.times(5).map((index) => (
+              <Grid item key={index}>
+                <ProjectListItemCard loading={loading} {...undefined} />
```

Comment:
This is why I thought it would be better to have a single data prop. Because those "required" fields aren't required if it's loading, but if we aren't loading then they are required. So if we don't have the data object then we are loading, else we have data.
This is a hack around TS that could easily be avoided.